### ISSUE TITLE
Genesis configuration "permission" field does not work

### ISSUE THREAD

**Issue by qmarliu** [xuperchain/xuperunion#623]
**Brief of the issue**
Configure permission field in xuper.json, But not working

**Your environment**
go1.13.5
gcc 7.4.0
os ubuntu18.04

**Repo steps**
1. Add the following configuration in xuper.json
"permission": {
            "CreateAccount" : { "rule" : "SIGN_THRESHOLD", "acl": {
                    "pm": {
                        "rule": 1,
                        "acceptValue": 1.0
                    },
                    "aksWeight": {
                        "dpzuVdosQrF2kmzumhVeFQZa1aYcdgFpN": 1.0
                    }
                }
            },
            "SetAccountAcl": { "rule" : "NULL", "acl": {}},
            "SetContractMethodAcl": { "rule" : "NU1LL", "acl": {}}
    }
2. Start node
3. still everyone can create contract account
**Expected behaviours**
only address "dpzuVdosQrF2kmzumhVeFQZa1aYcdgFpN" can create contract account

**What actually happened**
everyone can create contract account

**Comment by yucaowang** [comment]
Yes, system contracts' permission control is not implemented yet. The genesis config is preserved for it, we will implement in future, please stay tuned. Thanks!
